datasync/chaindatafetcher/kas/repository_transactions.go

```diff
@@ -0,0 +1,183 @@
+package kas
+
+import (
+	"fmt"
+	"github.com/klaytn/klaytn/blockchain"
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/common/hexutil"
+	"strings"
+)
+
+// TxFilteringTypes filters types which are only stored in KAS database.
+var TxFilteringTypes = map[types.TxType]bool{
+	types.TxTypeLegacyTransaction: true,
+
+	types.TxTypeValueTransfer:                      true,
+	types.TxTypeFeeDelegatedValueTransfer:          true,
+	types.TxTypeFeeDelegatedValueTransferWithRatio: true,
+
+	types.TxTypeValueTransferMemo:                      true,
+	types.TxTypeFeeDelegatedValueTransferMemo:          true,
+	types.TxTypeFeeDelegatedValueTransferMemoWithRatio: true,
+
+	types.TxTypeSmartContractDeploy:                      true,
+	types.TxTypeFeeDelegatedSmartContractDeploy:          true,
+	types.TxTypeFeeDelegatedSmartContractDeployWithRatio: true,
+
+	types.TxTypeSmartContractExecution:                      true,
+	types.TxTypeFeeDelegatedSmartContractExecution:          true,
+	types.TxTypeFeeDelegatedSmartContractExecutionWithRatio: true,
+}
+
+// transformToKasTxs transforms a chain event to transaction list according to the KAS database scheme.
+func transformToKasTxs(event blockchain.ChainEvent) []*Tx {
```

Comment:
This code is in `kas` repository. I think we don't need to add `kas` something.

Reply:
Right. I renamed the methods as well. 6c5074d74c98fafee0346dbc327c37015c5e4913